### PR TITLE
fix non-face image in age estimator model

### PR DESCRIPTION
**Pull request by I-Hong**
This PR fix the issue when Age estimator model adds non-face image feature.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -185,32 +185,36 @@ def gen():
...
+
+                for old_ix, new_ix in bbox_mapping:
+                    old_age = last_inference_image_ages[old_ix]
+                    new_age = age_results[new_ix]
+                    exp_decay_average_age = (new_age * EXP_DECAY_FACTOR
+                                             + old_age * (1.0 - EXP_DECAY_FACTOR))
+                    age_results[new_ix] = exp_decay_average_age
+
+                frames_since_update = 0
+            except:
```
Comment (bdwyer2) [inline app.py]: This is causing Travis to fail

```
$ flake8 . --max-line-length=127
./app.py:215:13: E722 do not use bare 'except'
```

**Comment by MLnick** [comment]
I think per https://github.com/IBM/MAX-Facial-Age-Estimator/pull/14#discussion_r261288314 we won't return `400` for no face found, but rather an empty response? So this approach will need to be adjusted here

**Comment by djalova** [comment]
I think the current response is fine. It will just return an empty array for "predictions."

File: app.py (hunk 1 above)
Comment (ajbozarth) [reply app.py]: For clarity, flake8 requires an except to catch an error value, not just run on an error

**Comment by MLnick** [comment]
@djalova yes that is what I meant - return an empty array.

File: app.py (hunk 1 above)
Comment (I-Hong) [reply app.py]: Thanks @bdwyer2 and @ajbozarth. I'll update this.

File: app.py [hunk 2]
```diff
@@ -185,32 +185,36 @@ def gen():
...
+                # Play back the video that has happened since the image was
+                # submitted for inference, updating the bounding boxes as we go
+                for img in images_since_submit:
+                    _, _ = tracker.update(img)
+
+                # Match faces from the previous match with the current match
+                bbox_mapping = match_bounding_boxes(last_inference_image_bounding_boxes,
+                                                    bounding_boxes)
+
+                for old_ix, new_ix in bbox_mapping:
```
Comment (bdwyer2) [inline app.py]: example

**Pull request closed without merging**